gae/room/templates/site_view.html

```diff
@@ -119,11 +119,25 @@
       <td>
         <div class="big jeditable"
 	           id="announcement_subject"
-	           name="{{ site.key.integer_id() }}">{{ site.announcement_subject }}</div>
+	           name="{{ site.key.integer_id() }}">{{ site.announcement_subject }}
+        </div>
```

Comment:
It looks funny, but I believe these </div> tags belong on the line above. Because otherwise it's introducing whitespace (a newline and some spaces) in the div contents, and that doesn't play well with something - either the jeditable thing or the <pre> formatting, I don't recall the details.